environments/kubernetes/kube-thanos.libsonnet

```diff
@@ -162,7 +167,7 @@ local rolebinding = k.rbac.v1.roleBinding;
                 ],
 
                 local volume = sts.mixin.spec.template.spec.volumesType,
-                volumes+: [
+                volumes: [
```

Comment:
so this means we override volumes, right?

Reply:
Yes, exactly.

Reply:
Exactly. We now completely replace whatever the volumes is. We actually need to do this now, as `null + []` doesn't compile. :upside_down_face: